docs/tutorials/field_type/7_add_content_and_edit_views.md

```diff
@@ -1,275 +1,116 @@
 # Add content and edit views
 
-To able to add and edit Content Item with our Field Type using PlatformUI, we need to create an edit view.
-The content view will also be needed to display correct information when viewing the Content Item in the "Content structure" tab in PlatformUI.
+To able to add and edit Content Item with our Field Type using PlatformUI, we will implement a FormMapper with FieldValueFormMapperInterface. 
```

Comment:
Are we still calling it PlatformUI? @DominikaK

Reply:
Good catch, it should be changed to `the Back Office`